_posts/2016-12-19-devlab-v3-is-here.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Devlab v3 Is Here"
+date: 2015-12-19 8:00:00
+categories: docker devlab development
+author: kent.safranski
+cover: /assets/images/covers/container-load.jpg
+description: After a year of dogfooding, a complete rebuild, and 110+ commits, we are excited to release Devlab v3.
+comments: true
+---
+
+After a year of dogfooding, a complete rebuild, and 110+ commits, we are excited to release Devlab v3.
+
+## New to Devlab?
+
+Devlab is a CLI tool that allows you to easily spin up [Docker](https://www.docker.com/) containers with all the services, environment variables, and settings you need to run a project in a "cleanroom".
```

Comment:
Could probably cut the quotes around "cleanroom" to add more of a punch to it, especially since it's in the leading section. Fairly minor, though.